src/test/groovy/com/jvm_bloggers/core/mailing/sender/EmailSendingSchedulerSpec.groovy

```diff
@@ -10,14 +10,14 @@ import spock.lang.Subject
 
 import java.time.LocalDateTime
 
+@Subject(EmailSendingScheduler)
 class EmailSendingSchedulerSpec extends Specification {
 
-    LocalDateTime NOW = LocalDateTime.now();
+    private final static LocalDateTime NOW = LocalDateTime.now()
```

Comment:
Maybe only `static` would be enough?

Reply:
As this is used only in the context of this test IMHO it is better to narrow the visibility as much as possible.